source/load-balancers/tests/wait-queue-timeouts.yml

```diff
@@ -0,0 +1,77 @@
+description: wait queue timeout errors include details about checked out connections
```

Comment:
Both tests in this file can't be run in Go because we don't support the `waitQueueTimeoutMS` option. I added some code to mock support for this option to validate that these can pass when I wrote them, but it's since been removed. It would be very helpful if another driver could validate them.